automl/docs/index.rst

```diff
@@ -1,8 +1,16 @@
 .. include:: README.rst
 
 
-Api Reference
--------------
+v1 API Reference
+------------------------
+.. toctree::
+    :maxdepth: 2
+
+    gapic/v1/api
+    gapic/v1/types
+
+v1beta1 API Reference
+----------------------
```

Comment:
Does this need language like other "old API version" blocks?  See, for instance, the [scheduler docs](https://github.com/googleapis/google-cloud-python/blob/master/scheduler/docs/index.rst).